app/src/org/commcare/views/media/AudioPlaybackButton.java

```diff
@@ -260,4 +294,36 @@ private void pausePlaying() {
         currentState = MediaState.Paused;
         refreshAppearance();
     }
+
+    private void animateProgress(int milliPosition, int milliDuration) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.HONEYCOMB) {
+            ProgressBar progressBar = (ProgressBar)findViewById(R.id.circular_progress_bar);
+            int startPosition = 0;
+            int progressDuration = 500;
```

Comment:
So from reading the documentation for `ofInt` and `setDuration`, my interpretation is that you want to flip these around. The [documentation for ofInt](https://developer.android.com/reference/android/animation/ObjectAnimator.html#ofInt(T, android.util.Property<T, java.lang.Integer>, int...)) says that `values` is "A set of values that the animation will animate between over time." While the [documentation for setDuration](https://developer.android.com/reference/android/animation/ObjectAnimator.html#setDuration(long)) says that `duration` is "The length of the animation, in milliseconds." Would you take the same conclusion from that?